examples/model_selection/plot_grid_search_stats.py

```diff
@@ -0,0 +1,548 @@
+# the curve from minus infinity to zero.
+
+better_prob = t_post.sf(0)
+
+print(f"Probability of {model_scores.index[0]} being more accurate than "
+      f"{model_scores.index[1]}: {better_prob:.3f}")
+print(f"Probability of {model_scores.index[1]} being more accurate than "
+      f"{model_scores.index[0]}: {1 - better_prob:.3f}")
+
+# %%
+# In contrast with the frequentist approach, we can compute the probability
+# that one model is better than the other.
+#
+# Note that we obtained similar results as those in the frequentist approach.
+# Given our choice of priors we are in essence performing the same
+# computations, but we are allowed to make different assertions.
+
+# %%
+# Sometimes we are interested in determining the probabilities that our models
+# have an equivalent performance, where "equivalent" is defined in a practical
+# way. A default approach [4]_ is to define estimators as practically
+# equivalent when they differ by less than 1% in their accuracy. But we could
+# also define this practical equivalence taking into account the problem we are
+# trying to solve. E.g. a difference of 5% in accuracy would mean an increase
+# of $1000 in sales, and we consider any quantity above that as relevant for
+# our business.
+#
+# In this example we are going to follow the suggestion in [4]_ and define the
+# Region of Practical Equivalence (ROPE) to be [-0.01, 0.01]. That is, we will
+# consider two models as practically equivelent if they differ by less than 1%
+# in their performance.
+#
+# To compute the probabilities of the classifiers being practically equivalent,
+# we calculate the area under the curve of the posterior over the ROPE
+# interval:
+
+rope_interval = [-0.01, 0.01]
+rope_prob = t_post.cdf(rope_interval[1]) - t_post.cdf(rope_interval[0])
+
+print(f"Probability of {model_scores.index[0]} and {model_scores.index[1]} "
+      f"being practically equivalent: {rope_prob:.3f}")
+
+# %%
+# We can plot how the posterior is distributed over the ROPE interval:
+
+x_rope = np.linspace(rope_interval[0], rope_interval[1], 100)
+
+plt.plot(x, t_post.pdf(x))
+plt.xticks(np.arange(-0.04, 0.06, 0.01))
+plt.vlines([-0.01, 0.01], ymin=0, ymax=(np.max(t_post.pdf(x)) + 1))
+plt.fill_between(x_rope, t_post.pdf(x_rope), 0, facecolor='blue', alpha=.2)
+plt.ylabel("Probability density")
+plt.xlabel("Mean difference")
+plt.title("Posterior distribution under the ROPE")
+plt.show()
+
+# %%
+# As suggested in [4]_, we can further interpret these probabilities using the
+# same criteria as the frequentist approach: Is the probability of falling
+# inside the ROPE bigger than 95% (alpha value of 5%)?  In that case we can
+# conclude that both models are practically equivalent.
+
+# %%
+# The Bayesian estimation approach also allows us to compute how uncertain we
+# are about our estimation of the difference. This can be calculated using
+# credible intervals which compute the range of values that have a particular
```

Comment:
Although in this case it is the same because we are talking about a normal distribution, I think it is not the equivalent to talk about a credible interval and the highest posterior density interval (see https://easystats.github.io/bayestestR/articles/credible_interval.html for example).

Reply:
IIUC, the reference only stated that a credible interval can be defined differently, e.g. HDI and ETI.
Explaining any of this would overload the text. So let's skip it.